### PR TITLE
Validate role/group dscription length

### PR DESCRIPTION
**Pull request by PanSpagetka**
https://issues.redhat.com/browse/RHCLOUD-9804

### CODE REVIEW

**Review by john-dupuy — CHANGES_REQUESTED** [review]
Checking this out locally I was able to verify that the name was checked and I was unable to click submit.

However, I enter in a really long name, RBAC will still crash -- I think because it is trying to make an API request to see if the name exists? If the text is over 150 characters, we shouldn't check that. 

I believe this is the behavior for groups, as I can't crash RBAC in a similar way when editing the name of a group.

**Comment by Hyperkid123** [comment]
> However, I enter in a really long name, RBAC will still crash -- I think because it is trying to make an API request to see if the name exists? If the text is over 150 characters, we shouldn't check that.

That would be the name async validation. We have to limit that aswell. Probably just don't trigger the call.

**Comment by PanSpagetka** [comment]
After last commit it shouldn't fire request if group name is too long

**Comment by john-dupuy** [comment]
@PanSpagetka ah, I meant we need to do that for roles, groups are already working properly!

**Review by john-dupuy — APPROVED** [review]
Testing out locally, I can no longer crash RBAC by putting in a long role name in the edit modal. Nice work @PanSpagetka :+1:

File: src/smart-components/role/add-role-new/schema.js [hunk 1]
```diff
@@ -66,6 +66,12 @@ export default (container) => ({
                 when: 'role-type',
                 is: 'create',
               },
+              validate: [
+                {
+                  type: 'max-length',
```
Comment (Hyperkid123) [inline src/smart-components/role/add-role-new/schema.js:71]: Can you use the constant as in the edit group modal?

**Pull request merged**